src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -180,9 +181,11 @@ public void onLoaded() {
                 Futures.addCallback(e.getCurrentExecutions(false), new FutureCallback<List<StepExecution>>() {
                     @Override
                     public void onSuccess(List<StepExecution> result) {
-                        LOGGER.log(FINE, "Will resume {0}", result);
-                        for (StepExecution se : result) {
-                            se.onResume();
+                        if (result != null) {
```

Comment:
Why would it be null?

Reply:
Or just use `@SuppressFBWarnings`?

Reply:
why suppress? if we don't want a null check shouldn't we just add an assertion that it isn't null

Reply:
Sure,

```java
assert result != null;
```

is also fine.